File: flower.py
import turtle


# Function to draw a petal
def draw_petal():
    turtle.color("red")
    turtle.begin_fill()
    turtle.circle(100, 60)
    turtle.left(120)
    turtle.circle(100, 60)
    turtle.left(120)
    turtle.end_fill()


# Function to draw the flower
def draw_flower():
    turtle.speed(10)
    turtle.bgcolor("white")

    # Draw the center of the flower
    turtle.color("yellow")
    turtle.begin_fill()
    turtle.circle(50)
    turtle.end_fill()

    # Draw the petals
    for _ in range(6):
        draw_petal()
        turtle.left(60)

    # Draw the stem
    turtle.color("green")
    turtle.width(10)
    turtle.right(90)
    turtle.forward(300)


# Set up the turtle
turtle.setup(800, 600)
turtle.penup()
turtle.goto(0, -50)
turtle.pendown()

# Draw the flower
draw_flower()

# Hide the turtle and display the window
turtle.hideturtle()
turtle.done()

File: flower.c
#include <graphics.h>
#include <conio.h>

void drawFlower() {
    int gd = DETECT, gm;
    initgraph(&gd, &gm, "C:\\Turboc3\\BGI");

    // Draw the stem
    setcolor(GREEN);
    line(320, 400, 320, 300);

    // Draw the petals
    setcolor(RED);
    circle(320, 250, 50);
    circle(370, 250, 50);
    circle(270, 250, 50);
    circle(320, 200, 50);

    // Draw the center of the flower
    setcolor(YELLOW);
    setfillstyle(SOLID_FILL, YELLOW);
    fillellipse(320, 250, 30, 30);

    getch();
    closegraph();
}

int main() {
    drawFlower();
    return 0;
}
